test/e2e/constants.ts

```diff
@@ -9,6 +9,9 @@ export const LOCAL_NODE_PRIVATE_KEY =
 export const DEFAULT_FIXTURE_ACCOUNT =
   '0x5CfE73b6021E818B776b421B1c4Db2474086a7e1';
 
+export const DEFAULT_FIXTURE_ACCOUNT_SHOTENED =
```

Comment:
Should this be `DEFAULT_FIXTURE_ACCOUNT_SHORTENED` (added `R`)

Reply:
PR is updated